pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -831,8 +839,8 @@ func translatePercentToFractionalPercent(p *networking.Percent) *xdstype.Fractio
 // envoy.type.FractionalPercent instance.
 func translateIntegerToFractionalPercent(p int32) *xdstype.FractionalPercent {
 	return &xdstype.FractionalPercent{
-		Numerator:   uint32(p * 10000),
-		Denominator: xdstype.FractionalPercent_MILLION,
+		Numerator:   uint32(p),
+		Denominator: xdstype.FractionalPercent_HUNDRED,
```

Comment:
do you know why we even had this 1M in the first place?

Reply:
Not sure why it was done like that. But I think for integers HUNDRED is right